ci/release_setup.md

```diff
@@ -1,14 +1,42 @@
 # Automating releases for a new Knative repository
 
+**Note:** Throughout this document, MODULE is a Knative module name, e.g. `serving`
+or `eventing`.
+
 By using the release automation already in place, a new Knative repository can
 get nightly and official releases with little effort. All automated releases
 are monitored through [TestGrid](http://testgrid.knative.dev).
 
-* **Nightly releases** are built every night from HEAD on the master branch.
-  They are referenced by a date/commit label, in the form `vYYYYMMDD-<commit_short_hash>`.
+* **Nightly releases** are built every night between 2AM and 3 AM (PST), from
+  HEAD on the master branch. They are referenced by a date/commit label, in the
+  form `vYYYYMMDD-<commit_short_hash>`. The job status can be checked in the
+  `nightly` tab in the corresponding repository dashboard in TestGrid. Images
+  are published to the `gcr.io/knative-nightly/MODULE` registry and manifests to
+  the `knative-nightly/MODULE` GCS bucket.
+
 * **Versioned releases** are usually built against a branch in the repository.
   They are referenced by a *vX.Y.Z* label, and published in the *Releases* page
-  of the repository.
+  of the repository. Images are published to the `gcr.io/knative-release/MODULE`
+  registry and manifests to the `knative-releases/MODULE` GCS bucket.
+
+Versioned releases can be one of two kinds:
+
+* **Major versioned releases** are those with changes to the `X` or `Y` values in
```

Comment:
nit, should this one be `Minor`, and the other one `Patch`?

Reply:
Clarified.